packages/tokamak/contracts-bedrock/scripts/start-deploy.sh

```diff
@@ -0,0 +1,215 @@
+
+  source $environement
+
+  export chainID=$(jq '.l1ChainID' $configFilePath)
+  echo "ChainID: $chainID"
+
+  echo "Project root: $projectRoot"
+  echo "Current dir: $currentPWD"
+  echo "Config file path: $DEPLOY_CONFIG_PATH"
+  echo "Env file: $environment"
+
+  # checkScriptInput
+  # checkAccount
+}
+
+updateSystem() {
+  echo "Update system"
+  apt update
+  DEBIAN_FRONTEND=noninteractive apt install git curl wget cmake jq build-essential -y
+}
+
+# checkScriptInput() {
+#   echo "Checking input"
+# }
+
+# checkAccount() {
+#   echo "Verifying accounts"
+# }
+
+installDependency() {
+  installGo
+  installJSPackages
+  installFoundry
+}
+
+installGo() {
+  echo "Install Go"
+  export PATH=$PATH:/usr/local/go/bin
+  if ! go version &> /dev/null; then
+    echo "Installing Go..."
+    cd /root
+    wget https://go.dev/dl/go1.21.13.linux-amd64.tar.gz
+    tar xvzf go1.21.13.linux-amd64.tar.gz
+    mv go /usr/local
+    export PATH=$PATH:/usr/local/go/bin
+    echo PATH=$PATH:/usr/local/go/bin >> ~/.bashrc
+    echo "   Installed $(go version)"
+    cd $currentPWD
+  else
+    echo "   Found $(go version)"
+  fi
+}
+
+installJSPackages() {
+  curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
+  DEBIAN_FRONTEND=noninteractive apt install -y nodejs
+  DEBIAN_FRONTEND=noninteractive apt install -y npm
+  npm install -g pnpm
+  curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | gpg --dearmor | tee /usr/share/keyrings/yarnkey.gpg >/dev/null
+  echo "deb [signed-by=/usr/share/keyrings/yarnkey.gpg] https://dl.yarnpkg.com/debian stable main" | tee /etc/apt/sources.list.d/yarn.list
+  apt update && apt install yarn
+}
+
+installFoundry() {
+  echo "Install Foundry"
+  if ! forge --version &> /dev/null; then
+    curl -L https://foundry.paradigm.xyz | bash
+    mv /root/.foundry/bin/* /usr/bin/
+    source ~/.bashrc
+    foundryup
+    echo "export PATH=/root/.foundry/bin:$PATH" >> /root/.bashrc
+    source /root/.bashrc
+    export PATH=/root/.foundry/bin:$PATH
+    echo $PATH
+    echo "   Installed $(forge --version)"
+  else
+    echo "   Found $(forge --version)"
+  fi
+}
+
+buildSource() {
+  echo "Start buiding source code"
+  source ~/.bashrc
+  export PATH=$PATH:/usr/local/go/bin
+  cd $projectRoot
+  pnpm install
+  make submodules
+  make cannon-prestate
+  make op-node
+  cd $projectRoot/packages/tokamak/contracts-bedrock && pnpm build
+  cd $currentPWD
+}
+
+deployContracts() {
+  echo "Start deploying smart contracts"
+  echo $DEPLOY_CONFIG_PATH
+  export IMPL_SALT=$(openssl rand -hex 32)
+  cd $projectRoot/packages/tokamak/contracts-bedrock
+  forge script scripts/Deploy.s.sol:Deploy --private-key $GS_ADMIN_PRIVATE_KEY --broadcast --rpc-url $L1_RPC_URL --slow --legacy --non-interactive
+  cd $currentPWD
+}
+
+generateL2Genesis() {
+  echo "Generate L2 genesis"
+  deployResultFile=$projectRoot/packages/tokamak/contracts-bedrock/deployments/$(printf "%d-deploy.json" "$chainID")
```

Comment:
When I run the deploy command, sometimes the deploy result file is generated as artifact.json instead of {chainID}-deploy.json. 

Could you check this issue?

Reply:
Oh. i'm so sorry!!

```bash
echo $DEPLOYMENT_OUTFILE
deployments/artifact.json
```
The issue was due to an environment variable I had set earlier. 
Now that [this commit](https://github.com/tokamak-network/tokamak-thanos/pull/309/commits/97ffdf5a85cb1525ea2cef5f27b15b07e36e2d1e) is in place, there’s no need to worry. Thank you!